package.json

```diff
@@ -47,6 +47,7 @@
     "postcss-values-parser": "1.3.1",
     "remark-frontmatter": "1.1.0",
     "remark-parse": "4.0.0",
+    "resolve": "^1.5.0",
```

Comment:
`1.5.0` 😂

Reply:
Not again... (at least the script catches it :D)